keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -774,10 +825,10 @@ enhancement:
   Yes.
 
```

Comment:
Comment for Line 821 "Does enabling the feature change any default behavior".

Actually  - I think the answer should be "it doesn't".
Because technically, without any changes from the user (just cluster operator), pods will start to be scheduled on other nodes.
[This is supposed to be change for the better to reduce problematic cases, but there is a change.]

Reply:
"Does enabling the feature change any default behavior" currently says "No changes for existing storage driver deployments." which is the same as "it doesn't".

Or did you mean to say that the answer should be "It does change default behavior"?

This depends on the interpretation of "the feature". My "no changes" was based on enabling "the feature" in Kubernetes, without enabling it in a CSI driver. Only once it is also enabled in a CSI driver, then there will be a change.

I'll put both in so that it is clearer.